src/content/docs/style-guide/writing-docs/article-templates/branching-docs-guide.mdx

```diff
@@ -0,0 +1,254 @@
+
+## How it works [#how]
+
+The install docs live in the `install` directory. There is a single `.yaml` file that defines how and when specific content is swapped in and out of the doc. The specific pieces of content are represented as `.mdx` files.
+
+## Directory overview [#install-directory]
+
+The install framework content lives in `src/install`. 
+
+To create a new install doc, add a directory to `install`.
+
+Your directory structure needs to look like this:
+
+```
+install
+|--assets (used for install images)
+|--config (used for the install yaml files)
+   |--agent-config (used for the agent config yml files)
+|--specific-install (used for a specific install doc)
+   |--sub-directory1 (used for .mdx files that will be swapped in)
+   |--sub-directory2 (used for .mdx files that will be swapped in)
+   |--sub-directoryx (you can have as many sub-directories as you need)
+```
+
+## The branching YAML file [#yaml-overview]
+
+The config (yaml) file is a highly customizable way to generate a wide variety of custom, interactive content. Being a YAML file, it’s extremely important to pay attention to indents and to make sure that every piece of the YAML file is correct. 
+
+If you swap in .mdx content in your YAML file, the .mdx content needs to be local to your install doc directory (for example, Java). Although it might be tempting to link to content anywhere on the Docs site for the purposes of reuse, this will cause your doc to break.
+
+The config file is extremely flexible and customizable. 
+
+
+
+## A sample YAML file [#sample-yaml]
+
+```
+agentName: pasta
+ agentType: APM
+ title: 'How to cook some great pasta'
+ metaDescription: "Let's make Gordan Ramsey Proud"
+ introFilePath: 'src/install/pasta/intro.mdx'
+ agentConfigFilePath: 'src/install/config/agent-config/java-newrelic.yml'
+ appInfo: 
+   - optionType: diet
+     label: 'What is your dietary preference?'
+     placeholder: 'I eat...'
+     options: 
+       - value: 'omni'
+         displayName: 'everything!'
+         logo: 'docker'
+       - value: 'vegan'
+         displayName: 'everything but animal products'
+         logo: 'glassfish'
+
+ steps:
+   - filePath: 'src/install/pasta/appInfo.mdx' 
+   - filePath: 'src/install/pasta/boil.mdx'
+   - filePath: 'src/install/pasta/agentConfig.mdx'
+   - filePath: 'src/install/pasta/type/default.mdx'
+     overrides: 
+       - filePath: 'src/install/pasta/type/omni.mdx'
+         selectedOptions: 
+           - optionType: diet
+             options: 
+               - value: 'omni'
+       - filePath: 'src/install/pasta/type/vegan.mdx'
+         selectedOptions: 
+           - optionType: diet
+             options: 
+               - value: 'vegan'
+   - filePath: 'src/install/pasta/sauce.mdx'
+ whatsNextFilePath: 'src/install/pasta/whatsNext.mdx'
+```
+
+## YAML field name details [#yaml-fields]
+
```

Comment:
@paperclypse I think these definitions would be easier to read if they were in a table.

Reply:
I agree!